src/get-started/test-service.md

```diff
@@ -58,27 +58,26 @@ We cover both kinds of testing in separate sections below.
 
 ## Testing Transaction Logic
 
-Let’s create [`tests/tx_logic.rs`][tests-tx_logic.rs], a file which will
+Let’s create [`src/tx_tests.rs`][tests-tx_logic.rs], a file which will
```

Comment:
Since transaction logic tests are no longer *integration* tests, this should be reflected in the "Preparing for Integration Testing" section; the section itself should probably be renamed to "Preparing for Testing." Also, versions of crates at the end of the section should be updated.